docker-entrypoint.sh

```diff
@@ -14,6 +14,13 @@ if [[ $DOCKER_HOST == unix://* ]]; then
 	fi
 fi
 
+# Compute the DNS resolvers for use in the templates
+export RESOLVERS=$(awk '$1 == "nameserver" {print $2}' ORS=' ' /etc/resolv.conf | sed 's/ *$//g')
+if [ "x$RESOLVERS" = "x" ]; then
+    echo "Warning: unable to determine DNS resolvers for nginx" >&2
+    unset RESOLVERS
```

Comment:
It is for now, yes, but at some point Docker said they are going to switch to a different resolver system, and that could cause issues, although I imagine they will maintain backwards compatibility.